app-office/gnucash/gnucash-3.7.ebuild

```diff
@@ -35,15 +35,16 @@ RDEPEND="
 	dev-libs/icu:=
 	dev-libs/libxslt
 	aqbanking? (
-		>=net-libs/aqbanking-5[gtk,ofx?]
-		sys-libs/gwenhywfar[gtk]
+		>=net-libs/aqbanking-5[ofx?]
+		sys-libs/gwenhywfar
 		chipcard? ( sys-libs/libchipcard )
 	)
 	gnome-keyring? ( >=app-crypt/libsecret-0.18 )
 	gui? (
 		gnome-base/dconf
 		net-libs/webkit-gtk:4=
 		>=x11-libs/gtk+-3.14.0:3
+		aqbanking? ( sys-libs/gwenhywfar[gtk] )
```

Comment:
Given our recent discovery wrt gwenhywfar[gtk] conflict, don't you need to make it depend on >=4.20.2 here since this is the first version in Portage to provide gtk3?

Do we need to think about older gnucash versions doing the same?

Reply:
> Given our recent discovery wrt gwenhywfar[gtk] conflict, don't you need to make it depend on >=4.20.2 here since this is the first version in Portage to provide gtk3?

Yes, that makes sense.

> Do we need to think about older gnucash versions doing the same?

I don't think so.  `equery d -a gnucash` returns nothing.  Stabilizing gnucash-3.7 should solve everything.

Reply:
The file collision report was made with gnucash 3.6 though, `equery` can't tell us anything relevant there.

Reply:
> The file collision report was made with gnucash 3.6 though,

Tested with 3.5 and 3.6, if I read it right.

Any changes made to 3.5 and 3.6 would revbump them to 3.5-r1 and 3.6-r1, no?  Assuming 3.7 makes it into the gentoo tree, what utility would  3.5-r1 and 3.6-r1 offer that isn't fulfilled by 3.7?

Reply:
As far as I can see, that means unpacking the tarballs, this bundling goes back even to 3.2. gnucash should have never been bumped like that. Anyway; I'm going to address this in gwenhywfar-4.20.2 myself using a blocker, as your dep fix in gnucash is only addressing one aspect of bug 695386 but does not fix the file collision. But then @titanofold won't be able to stabilise anything before 3.7, and I'm not going to make any assumptions on his plans. For gnucash-3.7 this means it will need to be stabilised in lockstep with gwenhywfar-4.20.2.

Reply:
> on second thought: could you add a <gnucash-3.7 blocker to the gwenhywfar ebuild in your PR? if I did it now, I would be blocking all existing gnucash ebuilds in tree...

Sure.

Reply:
With a depend atom like `!!<gnucash-3.7[gui,aqbanking]`,` repoman -dx full` emits:

```
  dependency.syntax [fatal]     4
   sys-libs/gwenhywfar/gwenhywfar-4.20.2-r1.ebuild: DEPEND: Invalid atom (!!<gnucash-3.7[gui]), token 15
   sys-libs/gwenhywfar/gwenhywfar-4.20.2-r1.ebuild: RDEPEND: Invalid atom (!!<gnucash-3.7[gui]), token 15
   sys-libs/gwenhywfar/gwenhywfar-4.20.2-r1.ebuild: DEPEND: Invalid atom (!!<gnucash-3.7[gui]), token 15
   sys-libs/gwenhywfar/gwenhywfar-4.20.2-r1.ebuild: RDEPEND: Invalid atom (!!<gnucash-3.7[gui]), token 15
```

The syntax seems valid. It's used in _sys-libs/ldb/ldb-1.1.31_ with `!!<net-fs/samba-3.6.0[ldb]`

`--force`?

Reply:
I assume `gtk? ( !!<app-office/gnucash-3.7[gui,aqbanking] )` would be the correct expression.

Reply:
We don't need a hardblocker there. The build times of gwenhywfar or gnucash are not in danger, we only need to ensure that a build of `gwenhywfar-4.20.2[gtk]` will cause the old `gnucash[aqbanking]` stuff to be unmerged. I say old, and I omit `gui` as you can see, because this condition only works on 3.7 which is not concerned anyway. ;)

So, in `RDEPEND` only, `gtk? ( !<app-office/gnucash-3.7[aqbanking] )` should do the trick.

Reply:
> We don't need a hardblocker there. The build times of gwenhywfar or gnucash are not in danger,

OK. I would have assumed a parallel emerge could trigger the conflict.